js/bybit.js

```diff
@@ -2538,16 +3151,56 @@ module.exports = class bybit extends Exchange {
         //       "retMsg": "Success."
         //    }
         //
+        // Unified Margin
+        //
+        //     {
+        //         "retCode": 0,
+        //         "retMsg": "Success",
+        //         "result": {
+        //             "totalEquity": "112.21267421",
+        //             "accountIMRate": "0.6895",
+        //             "totalMarginBalance": "80.37711012",
+        //             "totalInitialMargin": "55.42180254",
+        //             "totalAvailableBalance": "24.95530758",
+        //             "accountMMRate": "0.0459",
+        //             "totalPerpUPL": "-16.69586570",
+        //             "totalWalletBalance": "97.07311619",
+        //             "totalMaintenanceMargin": "3.68580537",
+        //             "coin": [
+        //                 {
+        //                     "currencyCoin": "ETH",
+        //                     "availableToBorrow": "0.00000000",
+        //                     "borrowSize": "0.00000000",
+        //                     "bonus": "0.00000000",
+        //                     "accruedInterest": "0.00000000",
+        //                     "availableBalanceWithoutConvert": "0.00000000",
+        //                     "totalOrderIM": "",
+        //                     "equity": "0.00000000",
+        //                     "totalPositionMM": "",
+        //                     "usdValue": "0.00000000",
+        //                     "availableBalance": "0.02441165",
+        //                     "unrealisedPnl": "",
+        //                     "totalPositionIM": "",
+        //                     "marginBalanceWithoutConvert": "0.00000000",
+        //                     "walletBalance": "0.00000000",
+        //                     "cumRealisedPnl": "",
+        //                     "marginBalance": "0.07862610"
+        //                 }
+        //             ]
+        //         },
+        //         "time": 1657716037033
+        //     }
+        //
         const result = {
             'info': response,
         };
         const data = this.safeValue (response, 'result', {});
-        const balances = this.safeValue (data, 'balances');
+        const balances = this.safeValue2 (data, 'balances', 'coin');
         if (Array.isArray (balances)) {
             // spot balances
             for (let i = 0; i < balances.length; i++) {
                 const balance = balances[i];
-                const currencyId = this.safeString (balance, 'coin');
+                const currencyId = this.safeString2 (balance, 'coin', 'currencyCoin');
                 const code = this.safeCurrencyCode (currencyId);
                 const account = this.account ();
                 account['free'] = this.safeString (balance, 'availableBalance');
```

Comment:
question, can't we use these fields to calculate the `used` amount? 
![image](https://user-images.githubusercontent.com/43336371/201897274-d0cf7a03-2e1e-4667-a23e-6caa0d835efa.png)